source/cgame/obituaries.h

```diff
@@ -697,6 +697,7 @@
 "TESTED",
 "THREW",
 "TIMED OUT",
+"TIMMA'D",
```

Comment:
I think this would be more appropriate as a suicide prefix

Can also add shanks

Reply:
I nuked it out again